src/ripple/protocol/TxFlags.h

```diff
@@ -123,11 +123,19 @@ constexpr std::uint32_t tfRenew                            = 0x00010000;
 constexpr std::uint32_t tfClose                            = 0x00020000;
 constexpr std::uint32_t tfPayChanClaimMask = ~(tfUniversal | tfRenew | tfClose);
 
-// NFTokenMint flags:
+// NFT/CFT flags:
 constexpr std::uint32_t const tfBurnable                   = 0x00000001;
+constexpr std::uint32_t const tfTransferable               = 0x00000008;
+
+// NFTokenMint flags:
 constexpr std::uint32_t const tfOnlyXRP                    = 0x00000002;
 constexpr std::uint32_t const tfTrustLine                  = 0x00000004;
-constexpr std::uint32_t const tfTransferable               = 0x00000008;
+
+// CFTokenIssuanceCreate flags:
+constexpr std::uint32_t const tfCannotFreezeBalances       = 0x00000002;
+constexpr std::uint32_t const tfRequiresAuthorization      = 0x00000004;
+constexpr std::uint32_t const tfCanTrade                   = 0x00000010;
+constexpr std::uint32_t const tfCanEscrow                  = 0x00000020;
```

Comment:
is escrow supported in CFT in V1?

Reply:
> is escrow supported in CFT in V1?

I don't think so. Likewise, DEX support is likely not in-scope either (@ledhed2222 correct me if that's wrong).

But these flags are still useful, IMO. Per the spec, these flags can't change once they've been set as part of a `CFTokenIssuanceCreate` (this is so that token holders have their expectations set properly throughout the lifetime of the token). 

So, by allowing issuers to set them now, we allow issuer to avoid having to re-issue in the future to support some of these use-cases (e.g., Escrow, DEX, etc).

Reply:
exactly - what he said ;)